account-kit/core/src/actions/watchSolanaConnection.ts

```diff
@@ -0,0 +1,31 @@
+import type { AlchemyAccountsConfig, SolanaConnection } from "../types";
+
+/**
+ * Subscribe to changes to the solana connection for the id
+ *
+ * @example
+ * ```ts
```

Comment:
nit: can we add `twoslash` here
```suggestion
 * ```ts twoslash
```